pages/place/routers/amend.js

```diff
@@ -8,6 +8,16 @@ module.exports = settings => form(Object.assign({
   configure: (req, res, next) => {
     getSchemaWithNacwos(req, settings.schema || schema)
       .then(mappedSchema => {
+        if (req.model.restrictions) {
+          // changesToRestrictions needs to be present in the model,
+          // otherwise no changes detected when only changing restrictions
+          req.model.changesToRestrictions = req.model.restrictions;
+        } else {
+          // only show changes field if we already have restrictions
+          delete mappedSchema.restrictions;
+          delete mappedSchema.changesToRestrictions;
+        }
```

Comment:
personally I don't like using `delete` - mainly because if mappedSchema is undefined you get a whacky useless error about not being able to convert a non object to an object. Stupid JavaScript. `_.unset` is safer

Reply:
Or in that case, `mappedSchema = omit(mappedSchema, ...)`